tools/PbarPElasticScatteringEventGenerator.h

```diff
@@ -44,8 +44,8 @@ std::pair<TTree*, double> generateEvents(double momentum, unsigned int num_event
 
   TDatabasePDG *pdg = TDatabasePDG::Instance();
   double mass_proton = pdg->GetParticle(2212)->Mass();
-  double fPhiMin = 0.0;
-  double fPhiMax = 2.0 * TMath::Pi();
+  double fPhiMin = 0.94 * TMath::Pi();
+  double fPhiMax = 1.06 * TMath::Pi();
```

Comment:
These values have to stay full phi by default, otherwise the software wont work for Panda. So similar to the theta boundaries, these values have to be changed from the application.